docs/src/main/tut/docs/play/README.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: docs
+title: Play Framework
+permalink: /docs/play/
+---
+
+# Play Framework integration
+
+It is easy to a freestyle program as a result of a Play Framework Action with the _freestyle-play_ module. This module provides an implicit conversion from `FreeS[F, A] => Future[A]` which allows a user to define a Free program as the result of any Play Action that expects a Future as a response.
```

Comment:
This should probably be "It is easy to _use_ a freestyle ..."?

We should probably use either

-  an implicit conversion from `FreeS[F, A]` to `Future[A]` or
-  an implicit conversion `FreeS[F, A] => Future[A]`.

Should we replace the "Free" in "to define a Free program" with `FreeS` or freestyle?